src/main/java/com/timgroup/statsd/NonBlockingStatsDClient.java

```diff
@@ -316,7 +376,7 @@ public NonBlockingStatsDClient(final String prefix, final String hostname, final
      *     if the client could not be started
      */
     public NonBlockingStatsDClient(final String prefix, final String hostname, final int port, final int queueSize, int timeout, int bufferSize,
-            final String[] constantTags, final StatsDClientErrorHandler errorHandler) throws StatsDClientException {
+                                   final String[] constantTags, final StatsDClientErrorHandler errorHandler) throws StatsDClientException {
```

Comment:
I've left this in for now but if you confirm you'd like it removed I can do so. At present [some](https://github.com/DataDog/java-dogstatsd-client/blob/90771e04702180bf81a9a3a06caac71ea7353563/src/main/java/com/timgroup/statsd/NonBlockingStatsDClient.java#L248) [versions](https://github.com/DataDog/java-dogstatsd-client/blob/90771e04702180bf81a9a3a06caac71ea7353563/src/main/java/com/timgroup/statsd/NonBlockingStatsDClient.java#L279) of the [constructor](https://github.com/DataDog/java-dogstatsd-client/blob/90771e04702180bf81a9a3a06caac71ea7353563/src/main/java/com/timgroup/statsd/NonBlockingStatsDClient.java#L376) indent the second line of arguments such that the first arguments of the line are in the same position
```
public NonBlockingStatsDClient(final String firstArgLineOne, ...
                               final String firstArgLineTwo, ...
```
but then [others](https://github.com/DataDog/java-dogstatsd-client/blob/90771e04702180bf81a9a3a06caac71ea7353563/src/main/java/com/timgroup/statsd/NonBlockingStatsDClient.java#L314) [don't](https://github.com/DataDog/java-dogstatsd-client/blob/90771e04702180bf81a9a3a06caac71ea7353563/src/main/java/com/timgroup/statsd/NonBlockingStatsDClient.java#L343)
```
public NonBlockingStatsDClient(final String firstArgLineOne, ...
        final String firstArgLineTwo, ...
```
So the added space in this change makes them all consistently line up those first args since that's how the majority of them played. But again, glad to conform to whatever standard suits.

Reply:
Ahh, I hadn't seen this inconsistency, thanks for making them consistent. I think this is fine to leave as is!